src/openforms/prefill/tests/files/personen.yaml

```diff
@@ -0,0 +1 @@
+../../contrib/haalcentraal/tests/files/personen.yaml
```

Comment:
symlink?

Reply:
Yes, to reuse the same OAS that we use for testing in Haal Centraal... I thought it might be better than copying it over in this folder?

Reply:
yeah it was a question to see if I interpreted this correctly, this is fine!